### ISSUE TITLE
[ShowcaseStandAlone] - Useage

### ISSUE THREAD

**Issue by BushTrocity** [keepcalm/BukkitForge#489]
The plugin loads right and you can make a shop, but no one can use it. It just spams my console with this. It won't error out or anything just spams. I have all the permissions right and everything.

2013-02-11 05:19:25 [FINEST] [SCS] player: Bushido276 requested permission: scs.use has?: true
2013-02-11 05:19:25 [FINEST] [SCS] bukkit default perms. return val = true

**Comment by DerpgonCz** [comment]
Try using GroupManager instead of PEX - but by my own experience - avery permission plugin does not work at least for me

**Comment by enterprise12nx** [comment]
Yes i would use group manager, very easy to use with essentials etc

**Comment by BushTrocity** [comment]
I do use Group Manager i hate PEX.

**Comment by Mastergalen** [comment]
There is actually more to the error log:

2013-02-16 06:34:25 [FINEST] [Minecraft] [SCS] player: flamescape requested permission: scs.admin has?: false
2013-02-16 06:34:25 [FINEST] [Minecraft] [SCS] permissions != null.  return val = false
2013-02-16 06:34:25 [FINEST] [Minecraft] [SCS] player: flamescape requested permission: scs.manage has?: true
2013-02-16 06:34:25 [FINEST] [Minecraft] [SCS] permissions != null.  return val = true
2013-02-16 06:34:25 [FINEST] [Minecraft] [SCS] player: flamescape requested permission: scs.remove has?: true
2013-02-16 06:34:25 [FINEST] [Minecraft] [SCS] permissions != null.  return val = true
2013-02-16 06:34:25 [FINEST] [Minecraft] [SCS] player: flamescape requested permission: scs.use has?: true
2013-02-16 06:34:25 [FINEST] [Minecraft] [SCS] permissions != null.  return val = true
2013-02-16 06:34:25 [SEVERE] [BukkitForge] Could not pass event PlayerInteractEvent to ShowCaseStandalone v383
org.bukkit.event.EventException
    at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:393)
    at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:66)
    at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:482)
    at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:464)
    at keepcalm.mods.bukkit.forgeHandler.ForgeEventHandler.interactEvent(ForgeEventHandler.java:480)
    at net.minecraftforge.event.ASMEventHandler_78_ForgeEventHandler_interactEvent_PlayerInteractEvent.invoke(.dynamic)
    at net.minecraftforge.event.ASMEventHandler.invoke(ASMEventHandler.java:35)
    at net.minecraftforge.event.EventBus.post(EventBus.java:103)
    at net.minecraftforge.event.ForgeEventFactory.onPlayerInteract(ForgeEventFactory.java:34)
    at ir.a(ItemInWorldManager.java:389)
    at iv.a(NetServerHandler.java:578)
    at fk.a(SourceFile:58)
    at cg.b(TcpConnection.java:458)
    at iv.d(NetServerHandler.java:136)
    at iw.b(NetworkListenThread.java:57)
    at ht.b(SourceFile:30)
    at net.minecraft.server.MinecraftServer.r(MinecraftServer.java:703)
    at ho.r(DedicatedServer.java:269)
    at net.minecraft.server.MinecraftServer.q(MinecraftServer.java:599)
    at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:497)
    at fy.run(SourceFile:849)
Caused by: av: Some parts of the Craft API cannot presently be implemented in Forge. Sorry.
    at org.bukkit.craftbukkit.inventory.CraftInventory.getViewers(CraftInventory.java:399)
    at com.kellerkindt.scs.shops.Shop.checkInteractConditions(Shop.java:1022)
    at com.kellerkindt.scs.shops.BuyShop.interact(BuyShop.java:51)
    at com.kellerkindt.scs.shops.Shop.onInteract(Shop.java:1558)
    at com.kellerkindt.scs.listeners.PlayerListener.onPlayerInteract(PlayerListener.java:107)
    at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:391)
    ... 20 more

**Comment by dequis** [comment]
tl;dr

> Some parts of the Craft API cannot presently be implemented in Forge. Sorry
> org.bukkit.craftbukkit.inventory.CraftInventory.getViewers

Maybe we could change the title to reflect this
